lib/core/services/search.service.ts

```diff
@@ -32,26 +32,45 @@ export class SearchService {
                 private searchConfigurationService: SearchConfigurationService) {
     }
 
-    async getNodeQueryResults(term: string, options?: SearchOptions): Promise<NodePaging> {
-        const data = await this.apiService.getInstance().core.queriesApi.findNodes(term, options);
+    getNodeQueryResults(term: string, options?: SearchOptions): Observable<NodePaging> {
```

Comment:
Why it needs to be Observable? The method never streams anything, it's a 1-time operation here.

Reply:
Is a braking change change it in promise we need to ripristinate it as observable plus all the other services are observable so is a consistent choose